Makefile

```diff
@@ -79,21 +78,45 @@ run: export THREESCALE_DEBUG=1
 run: generate fmt vet manifests
 	$(GO) run ./main.go --zap-devel
 
+# find or download controller-gen
+# download controller-gen if necessary
+CONTROLLER_GEN=$(PROJECT_PATH)/bin/controller-gen
+$(CONTROLLER_GEN):
+	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0)
+
+.PHONY: controller-gen
+controller-gen: $(CONTROLLER_GEN)
+
+KUSTOMIZE=$(PROJECT_PATH)/bin/kustomize
+$(KUSTOMIZE):
+	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.5.4)
+
+.PHONY: kustomize
+kustomize: $(KUSTOMIZE)
+
+OPERATOR_SDK = $(PROJECT_PATH)/bin/operator-sdk
+$(OPERATOR_SDK):
+	curl -sSL https://github.com/operator-framework/operator-sdk/releases/download/v1.2.0/operator-sdk-v1.2.0-x86_64-linux-gnu -o $(OPERATOR_SDK)
```

Comment:
A note here: This will be architecture dependent. If other people are trying to use it from other architectures it will fail.

Reply:
Updated. 

It would be good if @krishvoor could test on `ppc64le` arch